<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogIBPSet</class>
 <widget class="QDialog" name="DialogIBPSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>304</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>zero calibration</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonIbp1Zero">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>IBP1 zero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonIbp2Zero">
          <property name="text">
           <string>IBP2 zero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonIbp3Zero">
          <property name="text">
           <string>IBP3 zero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonIbp4Zero">
          <property name="text">
           <string>IBP4 zero</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>calibration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonIbp3Cal">
          <property name="text">
           <string>IBP3 cal</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxIbp3Cal">
          <item>
           <property name="text">
            <string>80</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>160</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>240</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>320</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>mmHg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>mmHg</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBoxIbp4Cal">
          <item>
           <property name="text">
            <string>80</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>160</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>240</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>320</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>mmHg</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonIbp1Cal">
          <property name="text">
           <string>IBP1 cal</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonIbp2Cal">
          <property name="text">
           <string>IBP2 cal</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>mmHg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxIbp2Cal">
          <item>
           <property name="text">
            <string>80</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>160</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>240</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>320</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxIbp1Cal">
          <item>
           <property name="text">
            <string>80</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>160</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>240</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>320</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonIbp4Cal">
          <property name="text">
           <string>IBP4 cal</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxIbpSacle">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>0-150</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-25-25</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>IBP Scale</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>wave speed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxWaveSpeed">
         <item>
          <property name="text">
           <string>6.25mm/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12.5mm/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25mm/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50mm/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100mm/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>full speed</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogIBPSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogIBPSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
